### ISSUE TITLE
Sync Watch Status Auth Error

### ISSUE THREAD

**Issue by Spritzup** [blacktwin/JBOPS#415]
I've been setting up JBOPS again after moving away from Plex for a year or so.  I'm trying to configure the Sync Watch Status script, but can't get it to work and keep getting the following error in the logs.  I've tried running it directly from the CLI and from within Tautulli with the same error.  Any help or guidance would be appreciated.

```
	Tautulli Notifiers :: Script error:
    Traceback (most recent call last):
        File "/config/JBOPS/utility/sync_watch_status.py", line 75, in <module>
            PLEX_TOKEN = CONFIG.data['auth'].get('server_token')
                                      ~~~~~~~~~~~^^^^^^^^
    KeyError: 'auth'
```

**Comment by blacktwin** [comment]
Are you using a `config.ini` for PlexAPI?

**Comment by Spritzup** [comment]
Yes, it turns out I needed to create a plexapi.config.ini file in the root of the config directory.... took me forever to stumble across the solution (I likely missed it in the documentation somewhere).  Apologies for not responding back earlier.

That said though, if it's not listed in the docs that you made need to create plexapi.config.ini for some scripts, perhaps it could be added?

In either case, thank you for your hard work on this, I appreciate it :)
